editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -160,7 +168,8 @@ export default function CrashDetailsPage({
           />
         </Col>
       </Row>
-      <Row id="units">
+      <Row id="units" style={{ scrollMarginTop: "55px" }}>
```

Comment:
TIL `scrollMarginTop`! What do do you think about turning this into a css class?